acceptance/acceptance_test.go

```diff
@@ -96,6 +96,13 @@ func TestInprocessMode(t *testing.T) {
 }
 
 func testAccept(t *testing.T, inprocessMode bool, singleTest string) int {
+	parallelFlag := flag.Lookup("test.parallel")
+	if parallelFlag != nil {
+		t.Logf("GOMAXPROCS=%d, parallel=%s, parallel_default=%s", runtime.GOMAXPROCS(0), parallelFlag.Value.String(), parallelFlag.DefValue)
```

Comment:
What's the benefit? This is provided as input to the run command right?

Reply:
only parallel=%s is provided, others are inferred. This is unrelated, to this PR, but I was using this when benchmarking different parallel options. Can drop it though.